content/install_maintain/installation/installing-cluster.md

```diff
@@ -129,9 +117,16 @@ The following sections describe how to install and configure Cloudify Manager cl
 1. [PostgresSQL Database Cluster ] ({{< relref "install_maintain/installation/installing-cluster.md#postgresql-database-cluster" >}})  
 2. [RabbitMQ Cluster] ({{< relref "install_maintain/installation/installing-cluster.md#rabbitmq-cluster" >}})  
 3. [Cloudify Management Service] ({{< relref "install_maintain/installation/installing-cluster.md#cloudify-management-service" >}}) 
-4. [Management Service Load Balancer] ({{< relref "install_maintain/installation/installing-cluster.md#management-service-load-balancer" >}}) 
+4. [File Permissions]({{< relref "install_maintain/installation/installing-cluster.md#file-permissions" >}})
```

Comment:
well I went  thru @geokala guide.This step is needed for the status reporter.
in his guide its in the end. @geokala  can you replay?

Reply:
It shouldn't be needed. @mateumann : Why do we have the following line in the guide?
sudo chmod 0755 /home/centos && sudo chown cfyuser:cfyuser -R ~/.cloudify-test-ca/

Reply:
1. In (some?) CentOS installations `/home/centos` has `0700` permission thus `/home/centos/.cloudify-test-ca/*` is not readable for `cfyuser`
2. Also as far as I can remember after `cfy_manager install` some files inside ~/.cloudify-test-ca/` ended up being owned by `root`

Reply:
That sounds like a bug. Can you raise a Jira for it, because we need to test that and we absolutely can't let that chmod+chown be part of the procedure.

Reply:
So remove this section?
@geokala

Reply:
Yes please, we'll make sure that it's not required.